security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -255,15 +260,15 @@ func (sc *SecretController) upsertSecret(saName, saNamespace string) {
 	// We retry several times when create secret to mitigate transient network failures.
 	for i := 0; i < secretCreationRetry; i++ {
 		_, err = sc.core.Secrets(saNamespace).Create(secret)
-		if err == nil {
+		if err == nil || errors.IsAlreadyExists(err) {
```

Comment:
nit:
Use log.Infof() in the case of errors.IsAlreadyExists(err).

Reply:
fixed